draft-ietf-mls-protocol.md

```diff
@@ -1314,10 +1314,9 @@ struct {
 } Extension;
 
 struct {
-    ProtocolVersion supported_version;
-    opaque client_init_key_id<0..255>;
```

Comment:
I am quite sure that it is needed for lookup, the emitter should have the ability to validate it and you want to include it in the signature.

Reply:
Why do you need it for lookup?  The only place it's referenced is Welcome, which references it by hash.

Reply:
Why do you need it for lookup?  The only place it's referenced is Welcome, which references it by hash.

Reply:
It might be fine to use the hash as an implicit identifier for outside the protocol for now but I suspect that in some applications we'll want the application to pick the identifier.